rockcraft/providers/_multipass.py

```diff
@@ -32,11 +32,16 @@
     get_managed_environment_snap_channel,
 )
 
-from ._buildd import BASE_TO_BUILDD_IMAGE_ALIAS, RockcraftBuilddBaseConfiguration
 from ._provider import Provider
 
 logger = logging.getLogger(__name__)
 
+BASE_TO_BUILDD_IMAGE_ALIAS = {
```

Comment:
Agreed on the duplication being a code smell.  The dictionary's final location will be `rockcraft/providers/providers.py`, so I'll move it there now.

This will be the first PR to use the new file `rockcraft/providers/providers.py`, so here's some context copied from https://github.com/canonical/charmcraft/pull/837.

<hr>
As the new `craft-providers` interface is built out, the directory `rockcraft/providers/` will be split into 2 places:

- `rockcraft/providers/providers.py` for `rockcraft`-specific code
- the `craft-providers` library

The code in the `rockcraft/providers/` directory will be split piece-by-piece, in a series of small PRs.

Once all the code is finally split, the file `rockcraft/providers/providers.py` will be moved to `rockcraft/providers.py`. Then, the directory will be removed. Finally, all references to the intermediate file `rockcraft/providers/providers.py` will be replaced with `rockcraft/providers.py`.